src/locales/fr-FR/translation.json

```diff
@@ -5,6 +5,12 @@
     "contactUs": "Un problème, une suggestion ? Contactez-nous à",
     "unknown_error": "Une erreur inattendue s'est produite, veuillez ré-essayer."
   },
+  "logout_dialog": {
+    "cancel": "Annuler",
+    "confirm": "Déconnecter",
+    "content": "Êtes-vous sûr de vouloir vous déconnecter de votre Cozy ?",
+    "title": "Déconnexion"
+  },
```

Comment:
Should we add the english version from cozy-home?

Reply:
We do not have english translations for anything as of now in the app, I'd guess this wouldn't be different